lib/web_ui/lib/src/engine/canvaskit/font_fallbacks.dart

```diff
@@ -120,7 +121,7 @@ class FontFallbackData {
     final List<SkFont> fonts = <SkFont>[];
     for (final String font in fontFamilies) {
       final List<SkFont>? typefacesForFamily =
-          skiaFontCollection.familyToFontMap[font];
+          (renderer.fontCollection as SkiaFontCollection).familyToFontMap[font];
```

Comment:
I believe getters are contravariant, so `CanvasKitRenderer.fontCollection` could be typed as `SkiaFontCollection`. With a specialized getter for `CanvasKitRenderer` this (and other places that do `as SkiaFontCollection`) could be just `canvasKitRenderer.fontCollection.familyToFontMap`.